tests/unit/fem/test_estimator.cpp

```diff
@@ -60,7 +60,8 @@ TEST_CASE("Least-squares ZZ estimator on 2D NCMesh", "[NCMesh]")
 {
    // Setup
    const auto order = GENERATE(1, 3, 5);
```

Comment:
Housekeeping in tests using `GENERATE`, it is helpful to include `CAPTURE(order);` as this will allow differentiation between failures for the parameterized tests, when looking at log files. Otherwise they're all conflated.

Reply:
Thanks for this trick. I did not know this. I will fix this one asap.